008-infinispan-getting-started/src/test/java/org/acme/quickstart/CacheResource.java

```diff
@@ -0,0 +1,39 @@
+package org.acme.quickstart;
+
+import io.quarkus.test.common.QuarkusTestResourceLifecycleManager;
+import org.infinispan.client.hotrod.RemoteCacheManager;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.wait.strategy.LogMessageWaitStrategy;
+import java.time.Duration;
+import java.util.Collections;
+import java.util.Map;
+
+public class CacheResource implements QuarkusTestResourceLifecycleManager {
+
+    private static GenericContainer INFINISPAN = null;
+    private static final Integer INFINISPAN_PORT = 11222;
+
+    private static RemoteCacheManager cacheManager;
+
+    @Override
+    public Map<String, String> start() {
+
+        INFINISPAN =
+                new GenericContainer("jboss/infinispan-server:10.1.6.Final-2")
+                        .waitingFor(new LogMessageWaitStrategy().withRegEx(".*Infinispan Server.*started in.*\\s"))
+                        .withStartupTimeout(Duration.ofMillis(20000))
+                        .withEnv("USER","my_username")
+                        .withEnv("PASS","my_password");
+
+        INFINISPAN.start();
+        final String hosts = INFINISPAN.getContainerIpAddress() + ":" + INFINISPAN.getMappedPort(INFINISPAN_PORT);
+
+        return Collections.singletonMap("quarkus.infinispan-client.server-list", hosts);
```

Comment:
is it intentional to define this property?

You have ` infinispan-client.hotrod.server-list=localhost:11222` in hotrod-client.properties

Reply:
Yes, otherwise it fails with connection refused. 
Line in hotrod-client.properties deleted, doing nothing.

Reply:
I was thinking the other way around.

Keep `infinispan-client.hotrod.server-list=localhost:11222` in hodrod-client properties

And return Collections.emptyMap() in this method.

I expect this should work like a charm

Reply:
@rsvoboda I tried exactly the same thing before I wrote the previous comment and the test failed.
I saw in logs that without `Collections.emptyMap()` it assigns IP address as 172.17.*.* i.e docker interface on my system, and when `Collections.emptyMap()` is used then throws error: connection refused 127.0.0.1:11222

Reply:
OK, so lets keep it as is